Dockerfile

```diff
@@ -1,16 +1,22 @@
-FROM docker.io/alpine:edge AS build
-WORKDIR /dms
-ADD . /dms
-RUN apk add --no-cache go gcc musl-dev
-RUN go mod tidy 
-RUN go build -trimpath -buildmode=pie -ldflags="-s -w" -o dms
+FROM docker.io/golang:1.19-alpine AS build
```

Comment:
I'd probably prefer just regular alpine unless there's a specific reason the language image is better. Mainly because I have a lot of multi-language projects and so these (and language devs) often seem fairly short-sighted about OS specifics.